generatedTypes/components/tabController/TabBar.d.ts

```diff
@@ -22,6 +26,10 @@ export interface TabControllerBarProps {
      * custom style for the selected indicator
      */
     indicatorStyle?: StyleProp<ViewStyle>;
+    /**
+     * Whether the indicator should be wide (as the item)
+     */
+    wideIndicator?: boolean;
```

Comment:
This prop shouldn't be a boolean IMO. 
Instead we better let the user decide on the indicator insets. 
This way the user has more control and if they want to have it wide they can pass 0

Reply:
Please see the changes here - https://github.com/wix/react-native-ui-lib/pull/1356